swarm/pss/client/client.go

```diff
@@ -17,6 +18,7 @@ import (
 )
 
 const (
+	rpcTimeout     = time.Second
```

Comment:
I we have a default timeout, we should give users the ability to change that timeout, but in my opinion there should not even be a default timeout. If the user needs a timeout, they should pass in a context which has the timeout they want to use.